neps/nep-0171.md

```diff
@@ -218,21 +218,21 @@ function nft_on_transfer(
 
```

Comment:
Shouldn't we have a changelog?

Reply:
Do you refer to the NEP changelog? There is a placeholder there already. We will fill it out once we have all the reviews, so we can compile all the feedback and the final decision details into it.

Reply:
Yes. wouldn't it be better to have it already set and be part of the final review?

Reply:
@robert-zaremba Ideally, yes, but I cannot see how to make it productive using GitHub UI. Feel free to review [my SME review comment](https://github.com/near/NEPs/pull/469#issuecomment-1476817621).